create_cert.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mkdir certs
+openssl req -newkey rsa:2048 -nodes -keyout certs/key.pem -x509 -days 365 -out certs/certificate.pem
```

Comment:
If you parameterize user may decide to generate cert with 4K/2K/1K, unless you have strong reason to limit this to 2K.

Reply:
Updated, thanks